index.js

```diff
@@ -7,16 +7,17 @@ function resize (gardrPluginApi) {
         var resizeVertical = !!item.options.resizeVertical;
         var container = item.options.container;
         var rendered = item.rendered;
+        var disableContainerResize = item.options.disableContainerResize || false;
```

Comment:
This could be named `enabled`  and flipped so the if statements are more readable.